CHANGELOG.md

```diff
@@ -43,6 +43,21 @@ Thank you to all who have contributed!
 Thank you to all who have contributed!
 - @<your-username>
 
+## [1.0.0-perf.1] - 2024-03-04
+
+This is a pre-release containing:
+- A new, experimental evaluator under `org.partiql.eval`.
+- Several breaking changes under `org.partiql.plan` and `org.partiql.types` and `org.partiql.spi`.
```

Comment:
Probably at some point before v1.0.0, we should list out all of the breaking changes.